April 2024 release blog post (#34)
ecamp/ecamp-site: src/content/posts/2024-04-release.de.md

---
title: April-Release
path: 2024-04-release
pubDate: 2024-04-14
description: Einladungen direkt akzeptieren, Block-Kategorien kopieren und mehr
image: '~/assets/images/080720_lotos_7896_small.jpg'
---

Bereits drei Wochen nach der letzten Aktualisierung schalten wir schon wieder ein Update auf eCamp v3 auf. Diesmal sind folgende Features dabei:

<div class="simple-columns bg-slate-100 dark:bg-slate-800">

![Einladungen im UI akzeptieren](~/assets/images/personal-invitations-de.png)

<div>

> Lagereinladungen können neu auch direkt auf der eCamp v3 Webseite akzeptiert werden, vorausgesetzt es wurde die E-Mailadresse eingeladen, welche zu deinem Account gehört. [#4865](https://github.com/ecamp/ecamp3/issues/4865){.issuelink} [#4932](https://github.com/ecamp/ecamp3/issues/4932){.issuelink}

Reviewer: ```suggestion
Lagereinladungen können neu auch direkt in eCamp v3 akzeptiert werden, vorausgesetzt die Einladung wurde an die E-Mailadresse deines Accounts geschickt. [#4865](https://github.com/ecamp/ecamp3/issues/4865){.issuelink} [#4932](https://github.com/ecamp/ecamp3/issues/4932){.issuelink}
```
